modules/ztp-image-based-upgrade-creating-backup-resources.adoc

```diff
@@ -165,7 +167,38 @@ $ oc apply -f oadp-cm-example.yaml
 
 .. If you are using GitOps ZTP, push the generated `ConfigMap` object to your git repository.
 
-. (Optional) To keep your custom catalog sources after the upgrade, add them to the `spec.extraManifest` in the `ImageBasedUpgrade` CR. For more information, see xref:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/operators/index#olm-catalogsource_olm-understanding-olm[Catalog source].
+. (Optional) To keep your custom catalog sources after the upgrade, generate another `ConfigMap` object for your catalog sources and add them to the `spec.extraManifest` in the `ImageBasedUpgrade` CR. For more information about catalog sources, see xref:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/operators/index#olm-catalogsource_olm-understanding-olm[Catalog source].
+
++
+--
+.Example CatalogSource CR
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  annotations:
+    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+  name: example-catalogsources
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  displayName: disconnected-redhat-operators
+  image: quay.io/example-org/example-catalog:v1
+----
+
+.Example configMapGenerator
+[source,yaml]
+----
+configMapGenerator:
```

Comment:
configMapGenerator requires a follow-up (creation of the configmap)

I see an example for it here:
https://69809--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_e-dge/ztp-image-based-upgrade#ztp-image-based-upgrade-shared-container-directory_ztp-image-based-upgrade


As an alternative - you can just show an example of creating the configmap.
Based on your example it's something like:
`oc create configmap  example-extra-manifests --from-file=99- example-extra-manifest.yaml=<path to yaml with Example CatalogSource CR>  -n openshift-lifecycle-agent`